CAN/PE3/ECU.h

```diff
@@ -0,0 +1,31 @@
+#ifndef ECU_h
+#define ECU_h
+
+#include "Arduino.h"
+
+class ECU {
+public:
+  ECU();
+  void update(unsigned long id, unsigned char msgBuf[], unsigned char msgLen);
+  void debugPrint(unsigned long id);
+  int rpm;
+  float tps;
+  float openTime;
+  float ignitionAngle;
+  float pressure;
+  bool pressureUnit;
```

Comment:
What are the options ? Not specified.

Reply:
I will change this to a tuple or something more descriptive.